ops/pipeline/test-jvm-gpu.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+## Test JVM packages with CUDA. Note: this script assumes that
+## the user has already built libxgboost4j.so with CUDA support
+## and place it in the lib/ directory.
+
+set -euox pipefail
+
+SCALA_VERSION=2.12
```

Comment:
is it intended to hardcode the scala version here?

Reply:
Yes, I chose to only test for 2.12 here.
@wbo4958 Should we test xgboost4j-spark-gpu for both Scala 2.12 and 2.13 ?

Reply:
Yeah, xgboost4j-spark-gpu should support both of 2.12 and 2.13

Reply:
Got it, I will add test for both 2.12 and 2.13.